test/extended/router/h2spec.go

```diff
@@ -153,63 +216,43 @@ func failingTests(testSuites []*h2spec.JUnitTestSuite) []h2specFailingTest {
 	return failures
 }
 
-func runConformanceTests(oc *exutil.CLI, t h2specRouteTypeTest) []*h2spec.JUnitTestSuite {
-	podName := "h2spec"
-	e2e.ExpectNoError(e2epod.WaitForPodNameRunningInNamespace(oc.KubeClient(), podName, oc.KubeFramework().Namespace.Name))
-
-	var results []*h2spec.JUnitTestSuite
+func runConformanceTests(oc *exutil.CLI, host, podName string) ([]*h2spec.JUnitTestSuite, error) {
+	g.By("Running the h2spec CLI test")
```

Comment:
Shouldn't podName be used in the By clause?  Just in case you want to use this func for some other podName?

Reply:
Otherwise, is there any need to pass in podName if it is always "h2spec"?